NAFAS.spec

```diff
@@ -7,7 +7,7 @@ nafas_version = "1.3"
 a = Analysis(['nafas/__main__.py'],
              pathex=['nafas'],
              binaries=[],
-             datas=[('nafas/sounds/*.wav', 'nafas/sounds')],
+             datas=[('nafas/sounds/silence.wav', 'nafas/sounds'), ('nafas/sounds/us1/*.wav', 'nafas/sounds/us1')],
```

Comment:
shouldn't we include all other speaker ids as well `us2`, `ch1`, etc.

Reply:
Currently, there is no control point for the speaker in `.exe` mode, so there is no need to add other speaker files.